src/main/java/uk/gov/hmcts/reform/prl/services/HelpWithFeesService.java

```diff
@@ -136,7 +136,7 @@ public Map<String, Object> setCaseStatus(CallbackRequest callbackRequest, String
                 caseDataUpdated.put(AWP_ADDTIONAL_APPLICATION_BUNDLE, additionalApplications);
                 caseDataUpdated.put(IS_THE_CASE_IN_DRAFT_STATE, YesOrNo.No.getDisplayedValue());
                 //WA fields
-                caseDataUpdated.put(AWP_WA_TASK_NAME, getAwpTaskName(additionalApplicationsBundle));
+                caseDataUpdated.put(AWP_WA_TASK_NAME, getAwpTaskName(additionalApplicationsBundle, true));
```

Comment:
@sealDebolina May I know why you have added this boolean flag ?

Reply:
removed, after discussion